### DISCUSSION TITLE
Istio multi-cluster eks

### DISCUSSION THREAD

**Discussion by said1995** [istio/istio#50723] [Q&A]
Hi Team,
Suppose I have two EKS clusters, both are deployed in different VPCs and these VPCs are peered together, that means each cluster pod can access other network pod by podip but not by service ip.

Cluster1 pods CIDR: 192.20.0.0/16
Cluster2 pods CIDR: 192.30.0.0/16

Cluster1 service CIDR: 10.0.0.0/16
Cluster2 service CIDR: 10.10.0.0/16

Here pod CIDR are from the VPC and hence VPCs are peered both cluster pods can communicate each other with podip.

Here service CIDRs are from different CIDR and which are internal to both of the clusters, so we cannot access the application hosted on the pod1 which is exposed by a service svc1 in cluster1 from the cluster2 using the svc1 ip address, but same can be accessed through the pod1 ip address.

In such a usecase what should be the deployment mode for istio multi cluster with multi primary? same network or different network?

Please let me know if I'm not clear and needs more information.

**Comment by rootsongjc** [comment]
You can check [Istio multi-network, multi-primary on EKS](https://github.com/aws-samples/istio-on-eks/tree/main/patterns/multi-cluster-multinetwork-multiprimary), I hope it helps.
